src/applications/vaos/appointment-list/components/cards/confirmed/ConfirmedAppointmentListItem.jsx

```diff
@@ -60,6 +60,7 @@ export default function ConfirmedAppointmentListItem({
   const isInPersonVAAppointment = !isVideo && !isCommunityCare;
   const isAtlas = isAtlasLocation(appointment);
   const videoKind = getVideoKind(appointment);
+  const isPhoneAppointment = appointment.legacyVAR.apiData.phoneOnly;
```

Comment:
Generally, I think we try to avoid pulling data from `legacyVAR.apiData` in the appointment list components. I think it would be better to put it in `appointment.vaos`, which can be done here: https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/vaos/services/appointment/transformers.js#L570

That `vaos` object is meant for info that we need, but doesn't really fit into the FHIR format.

Reply:
ahh, make sense. Done